docs/get-started/allocation/allocation-details.md

```diff
@@ -5,6 +5,34 @@ PIs and managers see the same allocation details as users, and can also add
 project users to the allocation, if they're not already on it, and remove users
 from an allocation.
 
+## How to View Resource Allocation Details in the Project
+
+A single project can have multiple allocations. To view details about a specific
+resource allocation, click on any of the available allocations in the Allocations
+section of the project details. Here, you can view the Resource Type, information
+about your Allocated Project, status, End Date of the allocation, and Actions
+button or any pending actions as shown below:
+
+![Veiwing Resource Allocation Details](images/viewing_resource_allocation_details.png)
+
+Clicking the Action icon for the corresponding allocation will open a page
```

Comment:
Clicking the Action icon -> Clicking the Action icon (it looks like a small folder, circled in the image above)
or something like that to make it clear what action Icon you are referring to.

Reply:
I will revise it to clarify:

> Clicking the Action icon (shown as a folder icon on the right side of each allocation, as seen in the image above)